docs/api.md

```diff
@@ -978,6 +988,11 @@ puppeteer.launch().then(async browser => {
 
 The method queries page for the selector. If there's no such element within the page, the method will resolve to `null`.
 
+#### frame.$$(selector)
+- `selector` <[string]> Selector to query page for
+- returns: <[Promise]<[Array]<[ElementHandle]>>> Promise which resolves to ElementHandles pointing to the page elements.
```

Comment:
I wonder, maybe all the `page` instances should be replaced by `frame` in `frame` API descriptions when appropriate?